src/test/scala/ai/privado/exporter/SourceExporterTest.scala

```diff
@@ -10,20 +10,17 @@ import io.shiftleft.semanticcpg.language.*
 import ai.privado.rule.{RuleInfoTestData, SinkRuleTestData}
 import ai.privado.dataflow.Dataflow
 import ai.privado.utility.StatsRecorder
+import ai.privado.exporter.{DataflowExporterValidator, SourceExporterValidator}
 
-class SourceExporterTest extends JavaFrontendTestSuite {
+class SourceExporterTest extends JavaFrontendTestSuite with DataflowExporterValidator with SourceExporterValidator {
 
   val ruleCache = RuleCache().setRule(
     RuleInfoTestData.rule
       .copy(sources = RuleInfoTestData.sourceRule, sinks = List(SinkRuleTestData.leakageKotlinRule))
   )
 
-  val auditCache    = AuditCache()
-  val privadoInput  = PrivadoInput(disableDeDuplication = true)
-  var dataflowCache = new DataFlowCache(privadoInput = privadoInput, auditCache = auditCache)
-  var appCache      = AppCache()
-
   "Identifier Tagger" should {
+    val privadoInput = PrivadoInput(disableDeDuplication = true)
```

Comment:
didn't understood the reason to use `disableDeDuplication = true` flag

Reply:
It was already there [here](https://github.com/Privado-Inc/privado-core/blob/ec46ab566bc18deb63ec920a7181b8f33adbb14c/src/test/scala/ai/privado/exporter/SourceExporterTest.scala#L44), this test was for not exporting derived sources into processing section ([commit](https://github.com/Privado-Inc/privado-core/pull/761/files#diff-1957173f36105a08d5f76bd7c22c3e5e205ed13795023a3eb0a9357af841a0e6)).